src/modules/buildingListFilter/i18n/fr_FR.root.json

```diff
@@ -3,55 +3,51 @@
 		{
 			"ids": [
 				0,
-				11,
-				13,
-				17
+				18
 			],
 			"name": "Base de pompiers"
 		},
 		{
 			"ids": [
 				2,
-				3,
-				6,
-				12,
-				14,
-				16
+				4,
+				5,
+				20
 			],
 			"name": "Secours"
 		},
 		{
 			"ids": [
-				5,
+				6,
 				8,
-				15
+				16,
+				19
 			],
 			"name": "Poste de police"
 		},
 		{
 			"ids": [
-				4,
-				7,
-				19
+				1,
+				3,
```

Comment:
remove the comma `,` in this line to fix build error

Reply:
thank I couldn't find the error